README.md

```diff
@@ -45,18 +39,28 @@ Heavy Forwarders | Yes | Conditional | Not required.
 \* **This add-on must be installed on either the HF or Indexers.**
 
 ## Input Requirements
-Set the sourcetype to "opnsense" in the inputs.conf file on the forwarder.
+Set the sourcetype to "opnsense" in the inputs.conf file on the forwarder, the actual data type will be automatically identified on ingestion.
 
 i.e.
 
 ```
 # Sample inputs.conf
 
-[monitor:///var/log/firewall.log]
+[monitor:///var/log/filter.log]
```

Comment:
If using a syslog server, this file could be named whatever and not filter or firewall. Which is why this is an example.

Reply:
OK cool, it wasn't clear on what was expected from the readme so went with UF, ignore this if you disagree but suggest linking to docs for syslog configuration for new users

also thanks so much for this TA - its fantastic

Reply:
Gotcha. I will try to make this a little more clear for new users. Thank you for your input on this.